index.html

```diff
@@ -151,1145 +151,1103 @@ <h4>Other Related Resources</h4>
+    </section>
 
-          <p>There are two possible types of closed segments:</p>
 
+    <section id="h_characters">
+      <h3>Characters</h3>
 
-          <ul>
-            <li><dfn>Single-Letter Closed Segment</dfn>, which contains only one letter that is in
-            its Isolated form.</li>
 
+      <p>Appendix <a href="#characters-tables"></a> lists characters used for the Arabic and
+      Persian languages. Characters used for these languages include letters and diacritics, three
+      sets of digits (usage depending on the region), punctuation (some common and some specific to
+      the script), symbols, and Unicode formatting characters.</p>
 
-            <li><dfn>Multi-Letter Closed Segment</dfn>, which contains more than one letter,
-            starting with an Initial form, zero or more Medial forms, and ending with a Final
-            form.</li>
-          </ul>
-        </section>
 
+      <p>The majority of these characters are common among different languages. There are three
+      different sets of digits used by different languages. Most of the alphabetical characters are
+      used by all the languages using Arabic scripts, but there are exceptions, such as the Arabic
+      letter <span class="lettername">yeh</span> being represented with two different characters,
+      <span class="uname">U+064A ARABIC LETTER YEH</span> (<span lang="ar" dir="rtl">ي</span>) and
+      <span class="uname">U+06CC ARABIC LETTER FARSI YEH</span> (<span lang="ar" dir=
+      "rtl">ی</span>). These differences among the character sets of each language are marked in
+      the appendix tables.</p>
 
-        <figure id="figure-closed_joining_segment_example">
-          <img height="140"
-            src="images/drawings/joining/joining-multiple-single-letter-closed-segments.png"
-            alt="A word with only single-letter closed segments.">
-          <img height="140"
-            src="images/drawings/joining/joining-single-multi-letter-closed-segment.png"
-            alt="A word that is just one long multi-letter closed segment.">
 
-          <figcaption>
-            Examples of closed joining segment types.
-          </figcaption>
-        </figure>
+      <p>Control characters are used to produce the correct spelling of the words or to ensure
+      correct combination with left-to-right content. Consequently, they should be preserved when
+      storing and displaying texts.</p>
+    </section>
 
 
-        <section id="h_open_joining_segments">
-          <h4>Open Joining Segments</h4>
+    <section id="h_direction">
+      <h3>Direction</h3>
 
 
-          <p>Under the certain cases, as noted in <a href="#joining_rule_3">Joining Rules 3</a>
-          <a href="#joining_rule_4">and 4</a>, <a>joining segments</a> can start with a
-          <a>join-to-right</a> form, or end with a <a>join-to-left</a> form, or both.</p>
+      <p>Arabic script is written from right to left. Numbers, even Arabic numbers, are written
```

Comment:
We early on decided to use the Unicode terminology to refer to numbers in order to minimise confusion.  So see https://w3c.github.io/alreq/index.html#h_preferred_terminology

Reply:
Yes, this is not my text but came up as new because of reorganisation.